SpotifyAPI.Web/Enums/SpotifyHttpStatusCode.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SpotifyAPI.Web.Enums
+{
+    //
+    // Summary:
+    //     Contains the values of status codes defined for Spotify specific uses
+    public enum SpotifyHttpStatusCode
+    {
+        TooManyRequests = 429
+    }
+}
```

Comment:
I'm not convided this class is needed. Spotify follows HTTPStatus code meanings, so I don't think we need this mapping.

Reply:
Gotcha.  I'll take another look.  Is 429 in HTTPStatus and I just missed it?